pgmpy/factors/continuous/ContinuousFactor.py

```diff
@@ -0,0 +1,109 @@
+import six
+
+
+class ContinuousFactor(object):
+    """
+    Base class for factors representing various multivariate
+    representations.
+    """
+    def __init__(self, variables, pdf, mean=None, covariance=None, lb=None, ub=None):
```

Comment:
@yashu-seth Why are you accepting mean, covariance, lb and ub in this. Those are parameters specific to JointGaussian